packages/bento-frontend/src/pages/dashTemplate/widget/WidgetView.js

```diff
@@ -65,9 +65,8 @@ const WidgetView = ({
             }}
             className={classes.customSwitch}
             disableRipple
-            onChange={() => {
-              themeChanger.toggleTheme();
-            }}
+            checked={themeChanger.dark}
+            onChange={themeChanger.toggleTheme}
```

Comment:
@amattu2 Quick question. Wouldn’t "onChange={themeChanger.toggleTheme}" will execute the "toggleTheme" function on load? Should we wrap "themeChanger.toggleTheme" in the arrow function?

Reply:
@iksheth Only if we add parenthesis at the end. This will just call the `toggleTheme` function when the switch changes.